content/ja/guides/hosting/iam/access-management/manage-organization.md

```diff
@@ -1,333 +1,323 @@
-{{% /alert %}}
+専用クラウドやセルフマネージド環境での SSO 設定については [SSO with OIDC]({{< relref path="../authentication/sso.md" lang="ja" >}}) や [SSO with LDAP]({{< relref path="../authentication/ldap.md" lang="ja" >}}) をご覧ください。{{% /alert %}}
 
-W&Bはデフォルトで自動プロビジョニングされたユーザーに「メンバー」役割を割り当てます。自動プロビジョニングされたユーザーの役割はいつでも変更可能です。
+自動プロビジョニングのユーザーにはデフォルトで "Member" ロールが割り当てられます。これらのロールはいつでも変更できます。
 
-Dedicated cloudインスタンスおよびSelf-managedデプロイメントでは、SSOを使ったユーザーの自動プロビジョニングがデフォルトでオンになっています。自動プロビジョニングをオフにすることができ、特定のユーザーを選んでW&B組織に追加することができます。
+専用クラウドやセルフマネージドデプロイメントでは SSO を利用した自動プロビジョニングがデフォルトで有効です。自動プロビジョニングは無効化も可能です。オフにすることで、特定ユーザーのみ手動で追加できるようになります。
 
-以下のタブでは、デプロイメントタイプに基づいてSSOをオフにする方法を説明します：
+以下のタブで、デプロイメントタイプごとに SSO をオフにする手順を説明します：
 
 {{< tabpane text=true >}}
-{{% tab header="Dedicated cloud" value="dedicated" %}}
-Dedicated cloudインスタンスを使用している場合で、自動プロビジョニングをオフにしたい場合は、W&Bチームに連絡してください。
+{{% tab header="専用クラウド" value="dedicated" %}}
+専用クラウドをご利用の場合、自動プロビジョニングをオフにするには W&B チームまでご連絡ください。
 {{% /tab %}}
 
-{{% tab header="Self-managed" value="self_managed" %}}
-W&Bコンソールを使用してSSOを使った自動プロビジョニングをオフにします。
+{{% tab header="セルフマネージド" value="self_managed" %}}
+W&B Console を利用して SSO の自動プロビジョニングをオフにできます：
 
-1. `https://<org-name>.io/console/settings/` に移動します。`<org-name>` をあなたの組織名に置き換えてください。
-2. **セキュリティ** を選択します。
-3. **SSOプロビジョニングを無効にする** を選択して、SSOを使った自動プロビジョニングをオフにします。
+1. `https://<org-name>.io/console/settings/` にアクセスします。`<org-name>` は組織名に変更してください。
+2. **Security** を選択
+3. **Disable SSO Provisioning** を選択し、自動プロビジョニングを停止します。
 
 {{% /tab %}}
 {{< /tabpane >}}
 
-{{% alert %}}
-
-SSOを使った自動プロビジョニングは、大規模にユーザーを組織に追加するのに役立ちます。なぜなら、組織の管理者が個別のユーザー招待を生成する必要がないからです。
+{{% alert title="" %}}
+SSO を利用した自動プロビジョニングは、組織管理者が個別にユーザー招待を発行しなくて済むため、大規模なユーザー追加に便利です。
 {{% /alert %}}
 
-### カスタム役割を作成する
+### カスタムロールの作成
 {{% alert %}}
-Dedicated cloudまたはSelf-managedデプロイメントでカスタム役割を作成または割り当てるには、エンタープライズライセンスが必要です。
+専用クラウドまたはセルフマネージド環境でカスタムロールの作成・割り当てには Enterprise ライセンスが必要です。
 {{% /alert %}}
 
-組織の管理者は、View-Only または Member 役割に基づいて新しい役割を作成し、追加の許可を追加することで詳細なアクセス制御を実現できます。チーム管理者は、チームメンバーにカスタム役割を割り当てることができます。カスタム役割は組織レベルで作成され、チームレベルで割り当てられます。
+組織管理者は Viewer または Member ロールをベースに、追加権限を組み合わせて独自のカスタムロールを作成し、より細かなアクセス制御を実現できます。Team 管理者は、チームメンバーにカスタムロールを割り当てられます。カスタムロールは組織レベルで作成し、Team レベルで割り当てます。
 
-カスタム役割を作成するには：
+カスタムロールを作成するには：
 
 {{< tabpane text=true >}}
-{{% tab header="Multi-tenant SaaS Cloud" value="saas" %}}
-1. https://wandb.ai/home に移動します。
-1. ページの右上隅にある **ユーザーメニュー** のドロップダウンを選択します。ドロップダウン内の **アカウント** セクションで **設定** を選択します。
-1. **役割** をクリックします。
-1. **カスタム役割** セクションで **役割を作成** をクリックします。
-1. 役割の名前を入力します。必要に応じて説明を追加します。
-1. カスタム役割のベースとする役割を **Viewer** または **Member** から選択します。
-1. 許可を追加するには、**許可を検索** フィールドをクリックし、追加する1つ以上の許可を選択します。
-1. 役割が持つ許可を要約している **カスタム役割の許可** セクションを確認します。
-1. **役割を作成** をクリックします。
+{{% tab header="マルチテナント SaaS Cloud" value="saas" %}}
+1. https://wandb.ai/home にアクセスします。
```

Comment:
I wonder why this list and the one a few lines down got re-numbered with explicit numbers.